Ical.Net/Evaluation/RecurrencePatternEvaluator.cs

```diff
@@ -7,45 +7,6 @@
 
 namespace Ical.Net.Evaluation
 {
-    /// <summary>
-    /// Much of this code comes from iCal4j, as Ben Fortuna has done an
-    /// excellent job with the recurrence pattern evaluation there.
-    ///
-    /// Here's the iCal4j license:
-    /// ==================
-    ///  iCal4j - License
-    ///  ==================
-    ///
-    /// Copyright (c) 2009, Ben Fortuna
-    /// All rights reserved.
-    ///
-    /// Redistribution and use in source and binary forms, with or without
-    /// modification, are permitted provided that the following conditions
-    /// are met:
-    ///
-    /// o Redistributions of source code must retain the above copyright
-    /// notice, this list of conditions and the following disclaimer.
-    ///
-    /// o Redistributions in binary form must reproduce the above copyright
-    /// notice, this list of conditions and the following disclaimer in the
-    /// documentation and/or other materials provided with the distribution.
-    ///
-    /// o Neither the name of Ben Fortuna nor the names of any other contributors
-    /// may be used to endorse or promote products derived from this software
-    /// without specific prior written permission.
-    ///
-    /// THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
-    /// "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
-    /// LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR
-    /// A PARTICULAR PURPOSE ARE DISCLAIMED.  IN NO EVENT SHALL THE COPYRIGHT OWNER OR
-    /// CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL,
-    /// EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO,
-    /// PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR
-    /// PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF
-    /// LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING
-    /// NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
-    /// SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
-    /// </summary>
```

Comment:
Comparing our code base with the current iCal4j code its hard to say wether there are similarities left. The mention goes back to 2009. Even more important is, that their BSD 3-Clause License is not compatible to our MIT License.